examples/bash_completion.py

```diff
@@ -0,0 +1,98 @@
+#!/usr/bin/env python3
+# coding=utf-8
+# PYTHON_ARGCOMPLETE_OK - This is required at the beginning of the file to enable argcomplete support
+"""A simple example demonstrating integration with argcomplete.
+
+This example demonstrates how to achieve automatic auto-completion of argparse arguments for a command-line utility
```

Comment:
@anselor Can you please review my expanded comment and make sure what I'm saying is actually accurate?  ;-)

Reply:
Looks good to be